c2corg_api/views/image.py

```diff
@@ -4,16 +4,17 @@
 from c2corg_api.models import DBSession
 from c2corg_api.models.document_history import has_been_created_by
 from c2corg_api.models.image import Image, schema_image, schema_update_image, \
-    schema_listing_image, IMAGE_TYPE, schema_create_image
+    schema_listing_image, IMAGE_TYPE, schema_create_image, \
+    schema_create_image_list
 from c2corg_common.fields_image import fields_image
 from cornice.resource import resource, view
 
 from c2corg_api.views.document import DocumentRest, make_validator_create, \
-    make_validator_update
+    make_validator_update, validate_document
 from c2corg_api.views import cors_policy, restricted_json_view
 from c2corg_api.views.validation import validate_id, validate_pagination, \
     validate_lang_param, validate_preferred_lang_param, \
-    validate_associations
+    validate_associations, _validate_associations
```

Comment:
If you need the function `_validate_associations` here, you should rename it to remove the private underscore marker (maybe to `validate_associations_for_document`?).